packages/components/src/Tabs/Tabs.css

```diff
@@ -20,25 +21,29 @@
   position: absolute;
   top: 0;
   bottom: 0;
-  width: 24px;
+  z-index: var(--elevation-base);
+  width: var(--space-large);
 }
 
 .overflowRight:after {
-  right: 0;
+  right: var(--public-tab--inset);
   box-shadow: inset -16px 0 16px -16px rgba(var(--color-black--rgb), 0.25);
 }
 
 .overflowLeft:before {
-  left: 0;
+  left: var(--public-tab--inset);
```

Comment:
Due to the padding on `.overFlow` the need to offset the box-shadow or else it will render in the wrong position and look incorrect